src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -178,4 +361,16 @@ public void withScalars() {
         );
     }
 
+    @Test
+    public void withCompositionOfScalars() {
+        MatcherAssert.assertThat(
+            new AvgOf(
+                () -> new MinOf(1.0d, 2.0d),
+                () -> new MaxOf(2.0d, 4.0d),
+                () -> new SumOf(1.0d, 2.0d, 2.0d),
+                new Ternary<>(true, 5.0, 1.0d)
```

Comment:
why just `5.0` without `d` or why `d` is need here at all?

Reply:
@neonailol typo. They were in the original code I inherited. I kept using the for the consistency. I can remove them if it does not correspond with the style of the library. :)